website/content/api-docs/auth/kubernetes.mdx

```diff
@@ -39,7 +39,7 @@ access the Kubernetes API.
   keys.
 - `issuer` `(string: "")` - Optional JWT issuer. If no issuer is specified, then this plugin will
```

Comment:
May want to document this param as being deprecated.

Reply:
Marked issuer and disable_iss_validation as deprecated.